packages/node_modules/@webex/webex-core/test/unit/spec/services/interceptors/server-error.js

```diff
@@ -52,28 +52,30 @@ describe('webex-core', () => {
                   developer: {
                     get: sinon.stub()
                   }
-                },
-                markUrlFailedAndGetNew: sinon.stub()
+                }
               },
               metrics: {
                 submitClientMetrics: sinon.spy()
+              },
+              services: {
+                markFailedUrl: sinon.stub()
               }
             }
           };
 
           get = interceptor.webex.internal.device.features.developer.get;
-          markUrlFailedAndGetNew = interceptor
+          markFailedUrl = interceptor
             .webex
             .internal
-            .device
-            .markUrlFailedAndGetNew;
+            .services
+            .markFailedUrl;
           submitClientMetrics = interceptor
             .webex
             .internal
             .metrics
             .submitClientMetrics;
```

Comment:
I'm currently fine with this being not "chained" and single-lined
I think this format of chaining should be reserved for functions (until prettier says otherwise)

Reply:
I'd prefer to keep the chaining as it will unnecessarily touch the scope of the lines below in the diff if that is implemented and exceed the line limit as a single line. Let me know if this is a must change.